apps/nextjs-website/src/components/PreHeader.tsx

```diff
@@ -1,12 +1,67 @@
 'use client';
 import React from 'react';
-import {
-  PreHeaderProps,
-  PreHeader as PreHeaderEC,
-} from '@pagopa/pagopa-editorial-components/dist/components/PreHeader';
+import { PreHeader as PreHeaderEC } from '@pagopa/pagopa-editorial-components/dist/components/PreHeader';
+import '@/styles/preHeader.css';
+import Icon from '@mui/material/Icon';
+import * as MuiIcons from '@mui/icons-material';
+import { PreHeader } from '@/lib/fetch/preHeader';
+// TODO: Component styles are loaded client-side (even if we move the import to a server side component above this one). This causes a flash of the unstyled component (even when everything is rendered as static HTML) that needs to be fixed.
 
-const PreHeader: React.FC<PreHeaderProps> = (preHeaderData) => (
-  <PreHeaderEC {...preHeaderData} />
+const formatValidMuiIcon = (input?: string | null): string | null => {
+  if (!input || !(input in MuiIcons)) {
+    return null;
+  }
+
+  // Convert from camel case (anExample) or pascal case (AnExample) to snake case (an_example)
+  return input
+    .replace(/(?:^|\.?)([A-Z])/g, (_, y) => `_${y.toLowerCase()}`)
+    .split(/^_/)
+    .join('');
+};
+
+const RefinePreHeaderProps = (props: PreHeader['data']['attributes']) => ({
+  ...props,
+  ...(props.leftCtas?.ctaButtons && {
+    leftCtas: {
+      ...props.leftCtas,
+      ctaButtons: props.leftCtas.ctaButtons.map((ctaBtn) => ({
+        ...ctaBtn,
+        disableRipple: true,
+        disableTouchRipple: true,
+        target: '_blank',
+        rel: 'noopener noreferrer',
+        sx: {
+          fontWeight: 'bold',
+          letterSpacing: '0',
+        },
+      })),
+    },
+  }),
+  ...(props.rightCtas?.ctaButtons && {
+    rightCtas: {
+      ...props.rightCtas,
+      ctaButtons: props.rightCtas.ctaButtons.map((ctaBtn) => ({
+        ...ctaBtn,
+        disableRipple: true,
+        disableTouchRipple: true,
+        target: '_blank',
+        rel: 'noopener noreferrer',
+        sx: {
+          fontWeight: '600',
+          letterSpacing: '.3px',
+        },
+        startIcon: <Icon>{formatValidMuiIcon(ctaBtn.icon)}</Icon>,
```

Comment:
Why are you add icon only on the right buttons?

Reply:
Because we are currently not expecting to have to render icons in the left buttons.

It does make sense to give that freedom for the future though. Updating right now.

Reply:
Something to note after the update.

Given that we're using MUI we have the possibility to also add an `endIcon` property which adds an icon to the right of text instead of to the left.

This sees no usage in the current design, it's not projected to and would add a field to the Strapi UI so I would not include it right this moment.

It may be something to consider in the future.